commons/data_access_layer/cosmos_db.py

```diff
@@ -277,15 +270,20 @@ def find_all(
             status_value = conditions.get('status')
             conditions.pop('status')
 
+        date_range = date_range if date_range else {}
+        date_range_params = (
+            self.generate_params(date_range) if date_range else []
```

Comment:
I consider , This should not be here.

Reply:
This method has its own ticket to attend its refactoring, please see https://ioetec.atlassian.net/browse/TT-245, nevertheless, I will take account of the suggestion.